notebooks/Name suggestion.ipynb

```diff
@@ -0,0 +1,1943 @@
+{
```

Comment:
<p>It seems we are duplicating a bit <code>utils.py</code>'s <code>gitbase_query</code>, probably we can merge both if we add the necessary logic to one of them!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
yep, I copy-pasted it from tensorflow analysis. But we can definitely use the same